src/osd/scrubber/scrub_backend.cc

```diff
@@ -230,37 +230,30 @@ void ScrubBackend::omap_checks()
   }
 
   stringstream wss;
+  const auto& smap = this_chunk->received_maps[m_pg_whoami];
```

Comment:
`operator[]` can create the pair it it doesn't exist. Easier to think when the read-only behavior is guaranteed. `at` is awkward but maybe `std::as_const(map)[key]` could do.

Reply:
Makes sense. We do know that the map is there (we would not have reached here unless build_map has finished its work), but I agree it's good to help the future reader.
I prefer 'at', as it's more idiomatic.